### ISSUE TITLE
Double header

### ISSUE THREAD

**Issue by nhyty** [newtheatre/history-project#342]
http://history.newtheatre.org.uk/years/07_08/crave/
http://history.newtheatre.org.uk/years/07_08/no_exit/

These were a double bill

**Comment by nhyty** [comment]
How are we going to deal with double headers like this, should it be one entry or two entries?

**Comment by wjdp** [comment]
Hmm, is a bit tricky. Probs should be combined, as per http://history.newtheatre.org.uk/years/12_13/the_proposal_the_bear/

General rule, if single poster then single record?

**Comment by wjdp** [comment]
By the way how dod we know these were a double bill? Do we have a poster?

**Comment by wjdp** [comment]
Also I'm rethinking the rule on this. If it was a compilation piece with same cast/crew in each segment then single record. If two shows that happened to run next to each other it should be separate. Is a bit of a judgement call on a case-by-case basis.

**Comment by nhyty** [comment]
poster

**Comment by wjdp** [comment]
Any obvious solution to Crave and No Exit?

**Comment by nhyty** [comment]
Not off the top of my head, we did this http://history.newtheatre.org.uk/years/12_13/the_proposal_the_bear/ for another one

**Comment by wjdp** [comment]
Yep, but I remember that and it was very much a single entity in terms of a 'show object'

**Comment by nhyty** [comment]
https://web.archive.org/web/20071104134634/http://www.newtheatre.org.uk/current.php

According to this there were two double headers this season

**Comment by nhyty** [comment]
going to be more troublesome give #487

**Comment by wjdp** [comment]
Canonical thing #486 looks to address

**Comment by samozzy** [comment]
Done some googling, here are some more known double bills until we know how we want to record this.
Whether that's adding it as a related show (#155) or a link within a trivia entry.
- 07/08 Crave / No Exit ([as above](https://github.com/newtheatre/history-project/issues/342#issue-115485933))
- 07/08 7 minutes 31 / Under the Blacklight
- 07/08 Under The Impression / The Third Condiment
- 07/08 The Box / Good Morning and Goodbye 
- 08/09 Offending the Audience / Nowhere Warm
- 12/13 The Proposal / The Bear ([as above](https://github.com/newtheatre/history-project/issues/342#issuecomment-157537558))
